package.json

```diff
@@ -8,7 +8,7 @@
     "build": "yarn run clean && webpack --mode production",
     "start": "webpack-dev-server --open --hot",
     "test": "jest --coverage",
-    "coveralls": "cat ./coverage/lcov.info | coveralls"
+    "coveralls": "jest --coverage && cat ./coverage/lcov.info | coveralls"
```

Comment:
this runs the test twice on Travis!
<img width="1377" alt="Screen Shot 2019-04-17 at 18 08 29" src="https://user-images.githubusercontent.com/20681465/56303246-d372ed80-613b-11e9-838c-9e2a1ed5c3ab.png">